Mathlib/Algebra/Category/ModuleCat/Sheaf/Abelian.lean

```diff
@@ -0,0 +1,29 @@
+/-
+Copyright (c) 2024 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.Algebra.Category.ModuleCat.Presheaf.Sheafification
+import Mathlib.CategoryTheory.Abelian.Transfer
+
+/-!
+# The category of sheaves of modules is abelian
+
+-/
+
+universe v v' u u'
+
+open CategoryTheory Limits
+
+variable {C : Type u'} [Category.{v'} C] {J : GrothendieckTopology C}
+
+namespace SheafOfModules
+
+variable (R : Sheaf J RingCat.{u}) [HasSheafify J AddCommGroupCat.{v}]
+  [J.WEqualsLocallyBijective AddCommGroupCat.{v}]
```

Comment:
Can you add a comment on what to import and what universes constraints there are for typical applications?

Reply:
Thanks for the suggestion. I have added the remark that if one imports `Algebra.Category.GroupCat.FilteredColimits`, then the `Abelian` instance is found automatically in the case of small categories.